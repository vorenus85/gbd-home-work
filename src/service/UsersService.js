export const UsersService = {
  getUsersData() {
    return [
      {
        id: 1,
        name: 'Tam치s Szab칩',
        email: 'tamas.szabo@gbd.hu',
        permission: 'admin'
      },
      {
        id: 2,
        name: 'Hajnalka Cs치sz치r',
        email: 'hajnalka.csaszar@gbd.hu',
        permission: 'admin'
      },
      {
        id: 3,
        name: 'Katy Walters',
        email: 'katy.walters@gbd.hu',
        permission: 'agent'
      },
      {
        id: 4,
        name: 'Ron Booker',
        email: 'ron.booker@gbd.hu',
        permission: 'agent'
      },
      {
        id: 5,
        name: 'Jordan Lewis',
        email: 'jordan.lewis@gbd.hu',
        permission: 'agent'
      },
      {
        id: 6,
        name: 'Pamela Williams',
        email: 'pamela.williams@gbd.hu',
        permission: 'agent'
      }
    ]
  },

  getUsers() {
    return Promise.resolve(this.getUsersData())
  }
}
