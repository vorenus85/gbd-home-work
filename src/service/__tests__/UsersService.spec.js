import { describe, it, expect } from 'vitest'
import { UsersService } from '@/service/UsersService'

describe('UsersService test', () => {
  it('should match snapshot', () => {
    const actualResult = () => UsersService.getUsers()

    expect(actualResult()).toMatchSnapshot()
  })
})
