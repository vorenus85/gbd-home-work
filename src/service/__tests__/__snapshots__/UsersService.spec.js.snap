// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UsersService test > should match snapshot 1`] = `Promise {}`;
